README.md

```diff
@@ -29,9 +29,8 @@ Note: Offline mode is removed in this fork, a server is required. You need to st
 - WASD to move forward, left, backward, right.
 - Space to jump.
 - Left Click to destroy a block.
-- Right Click or Cmd + Left Click to create a block.
+- Right Click or Ctrl + Left Click to create a block.
```

Comment:
Hm, look at this: (from [Wikipedia](https://en.wikipedia.org/wiki/Keyboard_layout#Modifier_keys))
>the `⌘ Cmd` key [is used] like the `Ctrl` on IBM PCs

what this concernes, you are right... but I'm not sure about this:
>The main use of the `Ctrl` key on Macs is to produce a secondary mouse click

Let's write both, because non-mac-users might have never heard about a "Cmd"-key :/